.ci/opensearch/Dockerfile.opensearch

```diff
@@ -5,9 +5,15 @@ ARG opensearch_path=/usr/share/opensearch
 ARG SECURE_INTEGRATION
 ENV SECURE_INTEGRATION=$SECURE_INTEGRATION
 
+# Starting in 2.12.0 security demo requires an initial admin password, which is set as myStrongPassword123!
+ENV password="admin"
+RUN if [ "$OPENSEARCH_VERSION" = "latest" ]; then \
+        export password="myStrongPassword123!"; \
+    fi
```

Comment:
This will break immediately after 2.12 is released.

Reply:
this should be updated to check for versions 2.12 and above